### ISSUE TITLE
gem-release load error on lower ruby version like 2.2.4

### ISSUE THREAD

**Issue by ForelaxX** [svenfuchs/gem-release#76]
After installing gem-release version 2.0.1, I found that all my gem commands print error message like this:

```
Error loading RubyGems plugin "~/.rvm/gems/ruby-2.2.4/gems/gem-release-2.0.1/lib/rubygems_plugin.rb": ~/.rvm/gems/ruby-2.2.4/gems/gem-release-2.0.1/lib/gem/release/cmds/release.rb:12: syntax error, unexpected tIDENTIFIER, expecting keyword_do or '{' or '('
...e given gemspec(s), pushes them to rubygems.org
```

Then I check the source code of the gem-release, seems like multi-line string syntax like  `<<~str` is not supported on ruby 2.2.4 or lower, syntax like `<<-str` works just fine.

I think it will be better if you can specific the ruby version gem-release supports in the gemspec or change all the `<<~str` to `<<-str.split("\n").map(&:lstrip).join("\n")`? Now I change my local gem-release code to make it work~

THX

**Comment by johnnyshields** [comment]
@svenfuchs this is happening for me too. May I suggest:
1) release gem-release version 2.0.2 which fixes this issue (use `<<-` instead of `<<~`) for all ruby versions
2) add the `<<~` back instead of `<<-` and release it as version 2.1.0

**Comment by johnnyshields** [comment]
@svenfuchs can you please help with this it's a major annoyance.

**Comment by PikachuEXE** [comment]
I just created #82 
Please have a look

**Comment by PikachuEXE** [comment]
If no rejection I will release #82 next week Friday

**Comment by PikachuEXE** [comment]
Fix released as [`2.0.3`](https://rubygems.org/gems/gem-release/versions/2.0.3)
